modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -41,10 +41,7 @@ apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: poison-pill-manager
-  namespace: poison-pill
-spec:
-  targetNamespaces:
-  - poison-pill
+  namespace: openshift-operators
```

Comment:
![image](https://user-images.githubusercontent.com/58551443/154982937-41c93b73-fd08-4001-901d-ebb466589317.png)
if you want to create PP in your own namespace, you need to create namespace, OperatorGroup and Subscription. And use the name of your namespace in "namespace" field. 
If you want to create PP in "openshift-operators" namespace, you don't need to create this namespace since it is already exists in openshift and also you don't need to create OperatorGroup for "openshift-operators" namespace.

Reply:
@prabinovRedhat Thank you. Can you verify and approve (lgtm)